scripts/reboot

```diff
@@ -30,6 +30,37 @@ function clear_warm_boot()
     /sbin/kexec -u || /bin/true
 }
 
+SCRIPT=$0
+
+function show_help_and_exit()
+{
+    echo "Usage ${SCRIPT} [options]"
+    echo "    Available options:"
+    echo "        -h -? : getting this help"
+    echo "        -x    : execute with -x flag"
```

Comment:
Another option `bash -x reboot`. Do you really need this as a parameter?

Reply:
Not 100% sure. Maybe someone wants to watch reboot process? Feel a bit wired only adding help :-)

Reply:
Removed -x now.